clients/python/test/e2e/test_notebooks.py

```diff
@@ -0,0 +1,33 @@
+import papermill as pm
+from pathlib import Path
+from tempfile import TemporaryDirectory
+import shutil
+import pytest
+from typing import List, Dict, Any
+import sys
+import osparc
+
+docs_dir:Path = Path(__file__).parent.parent.parent / 'docs'
+all_notebooks:List[Path] = list(docs_dir.glob("*.ipynb"))
+
+def test_notebook_config():
+    """ Checks the jupyter environment is configured correctly
+    """
+    config_test_nb:Path = Path(__file__).parent / 'data' / 'config_test.ipynb'
+    assert config_test_nb.is_file()
+    test_run_notebooks(config_test_nb,{'expected_python_bin':sys.executable, 'expected_osparc_version':str(osparc.__version__), 'expected_osparc_file':osparc.__file__})
+    assert len(all_notebooks) > 0, f'Did not find any notebooks in {docs_dir}'
+
+
+@pytest.mark.parametrize('notebook,params',list(zip(all_notebooks,[{}]*len(all_notebooks))))
```

Comment:
Qthe parametrization with papermill is something very nice but I wonder whether we need it right now?  Does it make any sense to add it as parameter now?

Reply:
There is only one reason why I did it this way: I want to make absolutely sure that I am using the same python binary (and therefore also `osparc` package) in the jupyter notebooks as the one I am running the tests with. Hence, I added a [this](https://github.com/bisgaard-itis/osparc-simcore-clients/blob/add-notebook-tests/clients/python/test/e2e/data/config_test.ipynb) test which is a parametrized notebook. The parameters I pass are simply the path to the python bin and `osparc.__file__`  and a few other things. That way I can check within the jupyter notebook, that I am running the expected python bin and the expected `osparc` package.